db/migrate/20200616220804_create_join_table_preserved_objects_complete_moabs.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJoinTablePreservedObjectsCompleteMoabs < ActiveRecord::Migration[6.0]
```

Comment:
I might expect the class name to match the table name. Are you open to naming the class `CreatePreservedObjectsPrimaryMoabs` and the table `preserved_objects_primary_moabs`? Or some such. Cc: @jmartin-sul

Reply:
Hi Mike - I was trying to use rails generator migration magic to create the table;  it didn't work for me -- I'm absolutely fine changing the class name.

Reply:
no strong feelings from me, but yeah, seems reasonable to align if it's not a pain